pkg/utils/utils.go

```diff
@@ -263,10 +263,14 @@ func configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetwor
 	}
 	// set PF mtu
 	if iface.Mtu > 0 && iface.Mtu != ifaceStatus.Mtu {
```

Comment:
looking at previous patch set, i see thats how you originally implemented it.
personally i prefer this way because its an allowed configuration and a resonable one. we should not emit warning to clutter an already cluttered  log.

Reply:
I have no preference either way. I thought having a log would make it easier for people to figure out that their MTUs aren't being applied. Otherwise users will need to read to code to know that this is the desired behavior.